newrelic.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['SDC SoundCloud Song Player'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: '438486c41c1545f42b0a40a65dfab212a17eNRAL',

```

Comment:
Talk to Leslie during our optional Q&A tonight. You'll probably end up needing to nix this New Relic key. I would recommend using environment variables to keep your future license key private. I've been using `require('dotenv').config()`.

Reply:
Still working on trying to get the first license key invalidated.